tests/test_methods.py

```diff
@@ -74,4 +75,30 @@ def test_reset():
     # reset the obseravtion environment
     new_ob_space, new_info = env.reset()
 
-    assert np.array_equal(new_ob_space, np.array([-1, -1]))
\ No newline at end of file
+    assert np.array_equal(new_ob_space, np.array([-1, -1]))
+
+# test reward function for both greenCrab and greenCrabSimplified
+def test_reward_func():
+    env = greenCrabSimplifiedEnv()
+    env.reset()
+    # self.state = self.self.init_state()
+
+    # test no trap laid for one timestep when no crab
+    action = np.array([-1, -1, -1])
+    assert env.reward_func(action) > 0 # are we expecting positive when no traps laid when no crabs
+
+    # test for all trap laid for one timestep when no crab
+    action = np.array([1, 1, 1])
+    assert env.reward_func(action) < 0 # are we expecting positive when no traps laid when no crabs
```

Comment:
Yeah, exactly! Though I think I got ahead of myself a bit since that change wouldn't fit the theme of the PR too well—let's come back to that afterwards!